wof/examples/flask/odm2/timeseries/runserver_odm2_timeseries.py

```diff
@@ -2,9 +2,6 @@
 
 import wof.flask
 
-curr_folder = os.path.abspath(os.path.dirname(__file__))
-sys.path.insert(0, '%s/../../../' % curr_folder)
```

Comment:
@ocefpaf That looks right, please see https://github.com/ODM2/WOFpy/blob/master/docs/WOFpyODM2LBRproduction_installation_notes.md#setting-up-runserver-script-and-wsgipy for a working sample of this script. Thanks.